src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -56,9 +55,9 @@ private static Workspace CreateWorkspaceWithProjectAndDocuments()
 
             Assert.True(workspace.TryApplyChanges(workspace.CurrentSolution
                 .AddProject(projectId, "proj1", "proj1.dll", LanguageNames.CSharp)
-                .AddDocument(DocumentId.CreateNewId(projectId), "goo.cs", "public class Goo { }")
-                .AddAdditionalDocument(DocumentId.CreateNewId(projectId), "add.txt", "text")
-                .AddAnalyzerConfigDocument(DocumentId.CreateNewId(projectId), "editorcfg", SourceText.From("config"), filePath: "/a/b")));
+                .AddDocument(DocumentId.CreateNewId(projectId), "goo.cs", SourceText.From("public class Goo { }", Encoding.UTF8, SourceHashAlgorithms.Default))
+                .AddAdditionalDocument(DocumentId.CreateNewId(projectId), "add.txt", SourceText.From("text", Encoding.UTF8, SourceHashAlgorithms.Default))
+                .AddAnalyzerConfigDocument(DocumentId.CreateNewId(projectId), "editorcfg", SourceText.From("config", Encoding.UTF8, SourceHashAlgorithms.Default), filePath: "/a/b")));
```

Comment:
Really continuing to not like this change being applied to tests -- we're just defaulting UTF8/SourceHashAlgoritms.Default so all the tests that don't care just got more verbose, and it obscures the tests that actually do care. 😢

Reply:
In this case we actually want to be explicit because the tests in this file that are testing checksum algorithm related APIs and reusing this helpers.